src/styles/utils/variables.scss

```diff
@@ -0,0 +1,6 @@
+$font-family: 'Roboto';
```

Comment:
use fallback fonts, check code style - fallback fonts

Reply:
modern-normalize.css will put backup fonts or not?
html {
	/* Improve consistency of default fonts in all browsers. (https://github.com/sindresorhus/modern-normalize/issues/3) */
	font-family:
		system-ui,
		'Segoe UI',
		Roboto,
		Helvetica,
		Arial,
		sans-serif
}